src/main/java/graphql/schema/DataFetcherFactoryEnvironment.java

```diff
@@ -25,7 +25,7 @@ public static Builder newDataFetchingFactoryEnvironment() {
         return new Builder();
     }
 
-    static class Builder {
+    public static class Builder {
```

Comment:
Ooops - that was a mistake and means no one outside Groovy could have created one